tensorflow_graphics/projects/mesh_rcnn/README.md

```diff
@@ -0,0 +1,3 @@
+# Mesh R-CNN 
+
+This project implements Mesh R-CNN with TensorFlow Graphics. 
```

Comment:
Add a link to the paper and github repo.

Describe this is a TF2 implementation of the github.